src/geo_traits_impl/geometry_collection.rs

```diff
@@ -0,0 +1,116 @@
+use super::GeometryCollectionType;
+use geo_traits::{Dimensions, GeometryTrait};
+
+impl geo_traits::GeometryCollectionTrait for GeometryCollectionType {
+    type T = f64;
+    type GeometryType<'b>
+        = &'b crate::Geometry
+    where
+        Self: 'b;
+
+    fn dim(&self) -> Dimensions {
+        self.geometry(0).unwrap().dim()
```

Comment:
Same, we should not crash on a valid geometry like `GEOMETRYCOLLECTION EMPTY`

Reply:
Addressed in https://github.com/georust/geojson/pull/245/commits/743ef4ea42d127cbeecdef536c5399b113e2d5f2a

Reply:
Addressed in https://github.com/georust/geojson/pull/245/commits/743ef4ea42d127cbeecdef536c5399b113e2d5f2a